src/test/java/io/strimzi/kafka/bridge/amqp/AmqpSinkBridgeEndpointMockTest.java

```diff
@@ -129,7 +130,7 @@ protected ProtonSender mockSender(ProtonQoS qos, String remoteAddress) {
         when(mockRecords.size()).thenReturn(1);
         // recordAt should not be called by the endpoint
         // but just to be sure...
-        when(mockRecords.recordAt(anyInt())).thenThrow(Exception.class);
+        when(mockRecords.recordAt(anyInt())).thenThrow(UncheckedIOException.class);
```

Comment:
Why this change?

Reply:
The mockito was complaining about `Checked Exception`.